src/main/java/org/icatproject/core/manager/EntityBeanManager.java

```diff
@@ -781,27 +782,37 @@ private void exportTable(String beanName, Set<Long> ids, OutputStream output,
 		}
 	}
 
-	private void filterReadAccess(List<ScoredEntityBaseBean> results, List<ScoredEntityBaseBean> allResults,
+	private void filterReadAccess(List<ScoredEntityBaseBean> acceptedResults, List<ScoredEntityBaseBean> newResults,
```

Comment:
Could you add add a method comment here please, in particular the relationship between acceptedResults and newResults. I realise there was no comment before but I like the fact that you have given the arguments better names and I think just a bit more explanation would help future developers.